Packs/iDefense/Integrations/FeediDefense/FeediDefense_description.md

```diff
@@ -0,0 +1,3 @@
+Fetch indicators from a iDefense feed. There are a million of indicators, consider using filters such as indicator severity, confidence, threat type and malware family. This feed fetches as default maximum of 50,000 indicators at once, so to reach all indicators you will need to fetch them in few rounds.
+ 
+**API Key** - To get the iDefense API token, see the iDefense documentation at <https://intelgraph.idefense.com/#/docs/view#page-section-2-0> under the "Obtain API token" section.
```

Comment:
```suggestion
Fetch indicators from a iDefense feed. 
As there are a million of indicators, consider using filters such as indicator severity, confidence, threat type and malware family.
 
**API Key** - To get the iDefense API token, see the iDefense documentation at <https://intelgraph.idefense.com/#/docs/view#page-section-2-0> under the "Obtain API token" section.
```

Reply:
So the description about how many indicators per fetch is not relevant? Cause I want the customer to know that he is not going to see all the indicators at one fetch

Reply:
I see your point. @kirbles19 / @ShirleyDenkberg Will help us curate something here in a couple of days.